### ISSUE TITLE
forms review

### ISSUE THREAD

**Issue by noahehall** [nirv-ai/uiproto#37]
in general check the form styles, aria tags, and the cursor type across labels, inputs, etc

especially take notice of the UX across form elements, there should be a holistic feel
